crash_similarity.py

```diff
@@ -15,10 +15,23 @@
 import download_data
 import utils
 
+import datetime
 import pyximport
 pyximport.install()
 
 
+# Checks if the model has been trained in the last 24 hours (using datetime.timedelta class)
+def check_training_time(time_interval):
```

Comment:
The implementation of this function could be greatly simplified: today's date - last training date > 1 day.